RoddyCore/src/de/dkfz/roddy/core/Analysis.java

```diff
@@ -496,6 +498,9 @@ private void abortStartedJobsOfContext(ExecutionContext context) {
             } catch (Exception ex) {
                 logger.severe("Could not successfully abort jobs.", ex);
             }
+        } else {
+            logger.severe("An workflow error occurred. However, strict mode is disabled and/or RollbackOnWorkflowError is disabled and therefore, all submitted jobs will be left running." +
```

Comment:
*A* workflow error ...

I like the warning, if things continue. Really seems to be a 180 degree turn in strategy. I am pretty sure this will be an improvement :-)